apps/dotcms-ui/src/app/view/components/dot-toolbar/dot-toolbar.component.scss

```diff
@@ -61,6 +72,16 @@ dot-toolbar-notifications {
     }
 }
 
+.toolbar__logo--whitelabel {
+    left: 0;
+    position: relative;
+    z-index: 1;
+    display: block;
+    width: 90px;
+    height: 16px;
```

Comment:
Shouldn't this be for ALL logos?

Reply:
Why would that be? I mean, the Top Nav height is too small.